tests/HL7Test.php

```diff
@@ -64,8 +64,23 @@ public function hl7_properties_can_be_set_using_chained_methods(): void
             ->withRepetitionSeparator('`')
             ->withEscapeCharacter('=')
             ->withHL7Version('555.666')
+            ->withSegmentEndingFieldSeparator(false)
             ->createMessage();
+        /**
+         * Tests:
+         *  - Field separator
+         *  - Component separator
+         *  - Subcomponent separator
+         *  - Repetition separator
+         *  - Escape character
+         */
```

Comment:
This comment (and the one below) may not be needed. The chained calls with readable method-names is clear enough for the reader about the expectation